Packs/AbnormalSecurity/ReleaseNotes/2_3_4.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### Abnormal Security Event Collector
+
+- Documentation improvements.
```

Comment:
Recommend removing the hyphen (bullet) for each since it's just one thing, also rephrase:

Improved documentation.